### PR TITLE
Fix ingress grpc

### PR DESCRIPTION
**Pull request by johnwc**
This PR fixes the issue with gRPC that occurs when using an ingress. I fully tested it using ingress controller from NGINX Inc. I have not been able to test it using the ingress from Kubernetes official, but the expected changes should be in place for it to work based on their documentation.

### CODE REVIEW

File: charts/pmm/templates/service.yaml [hunk 1]
```diff
@@ -21,3 +21,29 @@ spec:
   {{- end }}
   selector:
     {{- include "pmm.selectorLabels" . | nindent 4 }}
+{{- if .Values.ingress.enabled }}
```
Comment (denisok) [inline charts/pmm/templates/service.yaml:24]: should it be `{{- if $.Values.ingress.nginxInc }}` ?

**Comment by denisok** [comment]
Hi @johnwc,

Thank you very much for your contribution!

Could you please also update README ?

I am using https://github.com/bitnami-labs/readme-generator-for-helm to update tables, but you could do it manually.

Also you need to bump chart version to pass tests.

Thanks,
Denys

File: charts/pmm/templates/service.yaml (hunk 1 above)
Comment (johnwc) [reply charts/pmm/templates/service.yaml:24]: No, the service is only valid if ingress is enabled. Does not matter the ingress type.

**Review by denisok — APPROVED** [review]

**Comment by denisok** [comment]
@dnplkndll could you please review as well ?

**Comment by denisok** [comment]
@tplavcic @gen1us2k please review

**Review by gen1us2k — APPROVED** [review]

**Comment by gen1us2k** [comment]
Hi @johnwc,

Thanks for your contribution! Looks good to me

**Comment by Aleksa-del** [comment]
Hello, @johnwc ! Thank you for your contributions! We would like to send you a gift from Percona. Please, contact us at community-team@percona.com.

**Pull request merged**